### ISSUE TITLE
csv / xls(x) reading for more formats

### ISSUE THREAD

**Issue by JGuetschow** [pik-primap/primap2#36]
**Is your feature request related to a problem? Please describe.**

Currently reading from wide format csv files is implemented (long format under development). However, some datasets use different formats, e.g. a table with gas and category as dimensions where each year has an individual sheet or file. 

**Describe the solution you'd like**

Function that reads csv files from a folder or sheets from an xls(x) file into the interchange format.

**Describe alternatives you've considered**

Processing using external tools like the primap1 data_converter helper scripts

**Additional context**

For an example see e.g. the CEADs data for china. [www.ceads.net/](https://www.ceads.net/)

**Comment by JGuetschow** [comment]
Currently working on a set of function to help reading from different xls and csv formats which need custom code.

**Comment by JGuetschow** [comment]
The current way to tackle data reading is to use custom code to convert to a long or wide pandas DataFrame and then convert that to interchange format. It might not be worth the effort to further generalize the data reading functions as functions that can read from more format can become overly complex in use and might not be preferable over custom code per dataset.

**Comment by mikapfl** [comment]
Yeah, I'd say let's start with custom code per dataset, and if we notice we are writing the same code over and over, we generalize and include more into primap2.
